src/screens/connectors/LauncherView.js

```diff
@@ -104,6 +108,7 @@ class LauncherView extends Component {
     if (this.launcher.removeAllListener) {
       this.launcher.removeAllListener()
     }
+    cleanConnectorCookies(this.state.worker.url)
```

Comment:
Shouldn't this be `cleanAllConnectorCookies()` ?

Reply:
My aim was to only delete the cookies of the current connector URL, isn't that so? I only cleanAll() on reboot

Reply:
I think you're right though, it's far easier to remove everything on clear. And since we don't do parallel, I don't think it's an issue if we end up deleting something that should not be there anyway

Reply:
💀: after more testing, it seems I'm unable to selectively delete cookies on Android (either by deleting them, tampering with their value, or tampering with their expiration time). This is a HUGE blocker to this feature right now